src/drivers/usb/usb_pnp.c

```diff
@@ -77,15 +85,28 @@ void usb_dev_addr_settled(struct usb_dev *dev) {
 	usb_dev_configure(dev);
 }
 
+static int usb_add_newdev_action(struct lthread *self) {
+	struct usb_dev *dev;
+
+	dlist_foreach_entry(dev, &usb_newdev_list, dev_link) {
+		dlist_del(&dev->dev_link);
+		dlist_head_init(&dev->dev_link);
```

Comment:
Well, honestly, I don't quite understand the difference, although I won't insist on this. :)